tgui/packages/tgui/interfaces/GeneTek/tabs/ResearchTab.js

```diff
@@ -70,7 +71,7 @@ export const ResearchTab = (props, context) => {
             {ar.length ? ar.map(r => (
               <Section
                 key={r.ref}
-                title={r.name}
+                title={research[r.ref].name}
```

Comment:
Given that these are not the sort of things I would expect to change at runtime, we could also look to just have these statically added to the interface (increases filesize of the interface and means they can't be derived at runtime, but would reduce amount of data sent back and forth further).

Not requiring for this PR, possible future "optimization".

Reply:
Coupling the interface so tightly seems misstep for maintainability unless there was a way to enforce it at compile time or bake into the runtime checker.

I did consider it for some of the shared strings such as the unknown name and the In-progress and Unknown description but it felt somewhat clunky at the time.

Reply:
Yes, it may be over the top. I think there's a better way of doing it than what I said above, I'm mostly spitballing for the sake of "what if this doesn't optimize enough".